test/server/lib/MinIOExternalStorage.ts

```diff
@@ -0,0 +1,151 @@
+import * as minio from "minio";
+import sinon from "sinon";
+import * as stream from "node:stream";
+import {MinIOExternalStorage} from "app/server/lib/MinIOExternalStorage";
+import {expect} from "chai";
+
+describe("MinIOExternalStorage", function () {
```

Comment:
Could you add a `before` that skips the tests of MinIO is not configured? For example, in the `HostedStorageManager` test there is this:
https://github.com/gristlabs/grist-core/blob/f1e8cba57f02b1a24cb842923f6c261b868e4079/test/server/lib/HostedStorageManager.ts#L415-L418

Reply:
Do you think it's required? Minio is mocked so you don't have any client running (see `fakeClientClass`)